### DISCUSSION TITLE
Chainlink 2022 Hackathon!!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#1301] [Announcements]
This is your chance to put everything you've learned to the test, meet others, and more!

🎊🎊🎊 IT'S HERE 🎊🎊🎊

Web3 / Blockchain / Smart Contract Developers...

🔵 The Spring 2022 #Chainlink Hackathon is Here! 

🦾🦾🦾🦾🦾
AND.
ITS.
BIG.
🦾🦾🦾🦾🦾

Register and read here to see the event you don't want to miss...
https://chn.lk/springhackpc

⚽️ It kicks off April 22nd with an opening ceremony then followed by the BEST content the web3 space has to offer. 

🥳 These are MUST attend workshops, NO MATTER YOUR EXPERIENCE. To engage and ask questions with the best!

👶🏾 Brand new to web3? 
This is the hackathon for you.

👷Experience Engineer?
This is the hackathon for you.

🧙🏿‍♂️Wizard of Web3?
Yep. You guessed it. This is it.
👇

🪨The Core Hackathon workshops🪨
🗓 Saturday, April 23rd: 
📝 The purpose of Smart Contracts (Why are we here?)
👩‍💻 Introduction to Remix & Solidity (For those who are BRAND NEW!)
⛓ Introduction to Chainlink (To give you the power of hybrid smart contracts)

🗓 Sunday, April 24th: 
👷 Intro to HardhatHQ 
🍫 Intro to BrownieEth 
🌋 Intro to #Foundry
🧁 Intro to Truffle Suite 
⚓️ Intro to #Anchor w/ Solana 
Maybe an Apeworx & dapptools showing as well.

🗓 Monday, April 25th: 
🧪 Testing with HardhatHQ 
🧪 Testing with BrownieEth 

🗓 Tuesday, April 26th: 
🧪 Testing with #Foundry
🧪 Testing with Truffle Suite 
🧪 Testing with #Anchor w/ Solana 

💸 Then, we spice things up, with #defi and #nft tutorials!🎨
🗓 Wednesday, April 27th: 
💰 Code-along: DeFi Staking App
🎭 Code-along: NFT using IPFS 

🗓 Thursday, April 28th: 
🧑🏽‍🎨 Code-along: NFT 100% on-chain
👩🏻‍🔬 Code-along: DeFi Quant Engineering

🗓 Friday, April 29th: 
🤩 Code-along: Intro to Front End / Full Stack Web3
🎟 Code-along: Full Stack Nextjs Reactjs Solidity Raffle

🔵 Infrastructure tutorials too? YES 🔵 
🗓 Monday, May 2nd: 
⛓ Running a local Chainlink node
 🦾 Building and using External Adapters

🦿 Advanced 100x Web3 Developer Tutorials! ⚙️
🗓 Tuesday, May 3rd: 
⏫ Code-along: Upgradeable Smart Contracts
⛽️ EVM Opcodes & Gas Optimizations 
🗓️ Wednesday, May 4th: 
🏘️ Code-along: Build a DAO
🥩 Code-along: DeFi Staking App

🗓️ Thursday, May 5th: 
🔬 Security & Auditing 101
🏀 Code-along: Build a web3 game

🗓️ Friday, May 6th: 
🔬 Security & Auditing 201

And that's just a subset of all the amazing workshops, sponsor workshops will be added soon!
This is again, going to be one of the BIGGEST hackathons of the web3 space.

Want more reasons to join one of these? Check out my article on why you should join no matter what:
https://lnkd.in/ejJj7H6B

👯‍♀️ You excited? Us too!

Give this a retweet, a share, and follow so this hackathon can be the biggest one yet!

👯‍♀️👯‍♀️👯‍♀️👯‍♀️👯‍♀️👯‍♀️👯‍♀️👯‍♀️👯‍♀️
LOOKING FORWARD TO SEEING YOU THERE!
Register: https://chn.lk/springhackpc

**Comment by cromewar** [comment]
This is going to be a huge event, definitely will participate with a team!!

**Reply by PatrickAlphaC** [reply]
You got it!!

**Comment by mattjaf** [comment]
This is so cool. 

Been looking forward to this...

Definitely, hoping to at least get some swag out of it!

**Comment by shucks18** [comment]
Cant wait to see all of you guys on this it's gonna be an awesome event!

**Comment by Noveleader** [comment]
Got to participate in this, just awesome!!

**Comment by fontaipi** [comment]
Will participate for sure !

**Comment by eoskamiss** [comment]
Hi! I'm a beginner and I have registered for the hackathon. I got a welcome email and didn't hear from the organizers since. Am I missing any important instructions or updates? Can somebody share any links to instructions for uploading my project? Thanks and good luck !

**Reply by cromewar** [reply]
Hello @eoskamiss 
You can check the calendar here: https://chain.link/hackathon
Just add it to your own and join the conferences ;)
